crates/store/src/server/ntx_builder.rs

```diff
@@ -152,4 +154,55 @@ impl ntx_builder_server::NtxBuilder for StoreApi {
             next_token: next_page.token,
         }))
     }
+
+    #[instrument(
+        parent = None,
+        target = COMPONENT,
+        name = "store.ntx_builder_server.get_unconsumed_notes_for_network_account",
+        skip_all,
+        err
+    )]
+    async fn get_unconsumed_notes_for_network_account(
```

Comment:
@Mirko-von-Leipzig I'm trying to understand why the state has DB access and this functionality isn't required to be accessed via store fn like `crates/ntx-builder/src/store.rs`:
```rust
    /// Returns the list of unconsumed network notes.
    #[instrument(target = COMPONENT, name = "store.client.get_unconsumed_network_notes", skip_all, err)]
    pub async fn get_unconsumed_network_notes(&self) -> Result<Vec<NetworkNote>, StoreError> {
```

As in there is no corresponding `get_unconsumed_network_notes()` version of that ^ and it seems it is not required. In my head it would go `ntx-builder server -> store -> db` rather than `ntx-builder server -> state -> db`. Thanks in advance.

Reply:
> I'm assuming that server/ntx_builder.rs is confusing and sounds like its part of the ntx builder component?

Yea its this 👍